backend/src/apiserver/server/run_metric_util.go

```diff
@@ -24,7 +24,12 @@ import (
 )
 
 const (
-	metricNamePattern = "^[a-z]([-a-z0-9]{0,62}[a-z0-9])?$"
+	// This regex expresses the following constraints:
+	// * Allows lowercase/capital letters
```

Comment:
lowercase/uppercase just to be symmetric?

Reply:
good point, I forgot about uppercase. lol